src/redux/action-creators/registry/apply.ts

```diff
@@ -1,9 +1,9 @@
 import { Nos } from 'computable/dist/types'
-import { APPLY } from '../../constants'
+import { APPLY } from '../../../constants'
 import {
   FSA,
   Applicant,
-} from '../../interfaces'
+} from '../../../interfaces'
```

Comment:
i'm fine with that. 

however.

it can be done different ways. TS has allowed a way to do with with its config for some time now, but it introduces compilation problems. how do you propose doing it here? I'm not ok with it if it incurs more overhead than its worth

Reply:
I typically just add the whatever I want the root path to be, e.g. `src/`, to the `NODE_PATH` env var. I'm not yet familiar with the build process for the project, so I'm not sure if this will cause issues.